cmd/lncli/neutrino_active.go

```diff
@@ -225,6 +225,51 @@ func getCFilter(ctx *cli.Context) error {
 	return nil
 }
 
+var unbanPeerCommand = cli.Command{
+	Name:        "unbanPeer",
+	Usage:       "Unbans a previously banned peer and connects to it.",
+	Category:    "Neutrino",
+	Description: "Unban and connects to a previously banned peer",
+	ArgsUsage:   "address",
+	Action:      actionDecorator(unbanPeer),
+	Flags: []cli.Flag{
+		cli.BoolFlag{
+			Name: "permanent",
```

Comment:
maybe `permconnect` or something to make it more clear this has to do with how we connect to the peer?

Reply:
I will change to `permconnect`, thanks.